watertap/unit_models/tests/test_anaerobic_digestor.py

```diff
@@ -73,44 +55,9 @@
 
 
 # -----------------------------------------------------------------------------
-@pytest.mark.unit
-def test_config():
-    m = ConcreteModel()
-
-    m.fs = FlowsheetBlock(dynamic=False)
-
-    m.fs.props = ADM1ParameterBlock()
-    m.fs.props_vap = ADM1_vaporParameterBlock()
-    m.fs.rxn_props = ADM1ReactionParameterBlock(property_package=m.fs.props)
-
-    m.fs.unit = AD(
-        liquid_property_package=m.fs.props,
-        vapor_property_package=m.fs.props_vap,
-        reaction_package=m.fs.rxn_props,
-    )
-
-    assert len(m.fs.unit.config) == 16
-
-    assert not m.fs.unit.config.dynamic
-    assert not m.fs.unit.config.has_holdup
-    assert m.fs.unit.config.material_balance_type == MaterialBalanceType.useDefault
-    assert m.fs.unit.config.energy_balance_type == EnergyBalanceType.useDefault
-    assert m.fs.unit.config.momentum_balance_type == MomentumBalanceType.pressureTotal
-    assert not m.fs.unit.config.has_heat_transfer
-    assert not m.fs.unit.config.has_pressure_change
-    assert not m.fs.unit.config.has_equilibrium_reactions
-    assert not m.fs.unit.config.has_phase_equilibrium
-    assert not m.fs.unit.config.has_heat_of_reaction
-    assert not m.fs.unit.config.has_pressure_change
```

Comment:
The test harness no longer includes these types of tests, which is testing the default configuration arguments. I'm wondering if that is fine.

If a developer changed the default configuration arguments, then what would happen?
- the solution regression test will likely fail since now it is evaluating a different model
- the developer would then update the solution regression test
- the developer might not create a new test that covers the previous default options (I think this is the main issue). 

However, this could always happen, the developer could fail to test all configuration argument options. The only way we could prevent this test omission is if we somehow ensure all configuration arguments are tested. I'm not sure how we would do this, and its definitely out of scope for this PR. So I think this is fine.

Reply:
> The test harness no longer includes these types of tests, which is testing the default configuration arguments. I'm wondering if that is fine.
> 
> If a developer changed the default configuration arguments, then what would happen?
> 
>     * the solution regression test will likely fail since now it is evaluating a different model
> 
>     * the developer would then update the solution regression test
> 
>     * the developer might not create a new test that covers the previous default options (I think this is the main issue).
> 
> 
> However, this could always happen, the developer could fail to test all configuration argument options. The only way we could prevent this test omission is if we somehow ensure all configuration arguments are tested. I'm not sure how we would do this, and its definitely out of scope for this PR. So I think this is fine.

I think the answer here is we just need to look at the coverage reports to make sure all the configuration options are covered.